### ISSUE TITLE
Property List: Summary (Beta)

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#3785]

**Comment by RDmitchell** [comment]
It seems like Summary might be taking the place of the older Reports, although Reports allows more control over the fields displayed, and I don't think we need to have that functionality in 2 different places. I would vote for working on making Reports more robust and taking the graphs out of Summary, unless I am not understanding the role of Summary.

![image](https://user-images.githubusercontent.com/6314950/214433030-6a40cad7-1252-4903-9f9e-df22ca1cb001.png)

**Comment by RDmitchell** [comment]
This is going to be deprecated, and merged into the regular reports page.

**Comment by nllong** [comment]
> This is going to be deprecated, and merged into the regular reports page.

correct, summary page plots will move to the reports page. We are also thinking on moving to reports tab to the "insights" tab... but welcome to hear suggestions on that.

We need to figure out where to put the "column stats". Maybe that becomes an analysis and the result is a pretty-fied table?

**Comment by RDmitchell** [comment]
If Reports do get moved to Insights, I would hope that first we do a rework of the Insights UI, because I think now it is confusing to the users.

**Comment by nllong** [comment]
I think we are planning on leaving the Reports page the same just under the "insights" heading.

**Comment by RDmitchell** [comment]
Ah, I see. Yes, that probably makes sense !

**Comment by isalanglois** [comment]
Closing this ticket due to SCEP updates being made
